src/main/java/edu/flab/member/dto/MemberRankDto.java

```diff
@@ -10,5 +10,5 @@ public class MemberRankDto {
 	private final String profileUrl;
 	private final Integer judge_point;
 	private final String nickname;
-	private final RankTier rankTier;
+	private final LolTier lolTierGroup;
```

Comment:
```suggestion
	private final LolTier lolTier;
```

Group 이란 네이밍의 의미가 있을까요?

Reply:
이것저것 네이밍하다가 전에 작성해놨던걸 실수로 올렸네요. 정정해주셔서 감사합니다. ㅎㅎ